en/developer_guide/multi_panel.rst

```diff
@@ -0,0 +1,94 @@
+Add panel to an administration view
+===================================
+
+Context
+-------
+
+In Open Orchestra, if you want to edit an element, you can only display the full form. In some cases
+you want to embillish the edition by splitting the full form in some smaller one.
+
+Open Orchestra already provides you a way to `use a custom Backbone view`_.
+This solution may be possible but will require a large customization of the form view.
+
+To prevent you from doing this work, Open Orchestra provide a way to add multiple panel in your edition form.
+
+Panel System
+~~~~~~~~~~~~
+
+Each element facade contains a list of links that may be used to display more specific information about them.
```

Comment:
I think something lacks here : behind links there are some controllers dedicated to the render of the smaller parts of the main form.
So I think we should start the explanation by the conception of this subform controllers, then tell how to use it.